### PR TITLE
new breadcrumbs

### PR DESCRIPTION
**Pull request by pandafulmanda**
## Homework, due in future
screen shot 2015-04-29 at 3 16 05 am

## Homework, due in past, feedback is viewable
screen shot 2015-04-29 at 3 16 23 am

## iReading
screen shot 2015-04-29 at 3 16 35 am

### CODE REVIEW

**Comment by philschatz** [comment]
@Fredasaurus Any objections on including these in?

**Comment by Fredasaurus** [comment]
Go for it, though we may need to make some changes later.

File: style/icons/all.less [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
 // Note: Use tokens instead of strings to be able to do LESS compile-time validation
 .icon-feedback { .x-icon-bg(feedback); }
-.icon-homework { .x-icon-bg(homework); }
+.icon-exercise, .icon-homework, .icon-end { .x-icon-bg(homework); }
```
Comment (philschatz) [inline style/icons/all.less]: Can you place selectors on separate lines? Maybe this should be done in the JS that way it's easier to grep for when changing.

File: style/task-step.less [hunk 2]
```diff
@@ -14,13 +31,49 @@
...
+    font-weight: 500;
+    cursor: pointer;
+    margin: 2px;
+    display: inline-block;
+    text-align: center;
+    background: @white;
+    color: @tutor-light-gray;
+    .transition(all .1s ease-in);
+
+    &:hover, &.active {
```
Comment (philschatz) [inline style/task-step.less]: Can you put selectors on separate lines?

File: style/icons/all.less (hunk 1 above)
Comment (pandafulmanda) [inline style/icons/all.less]: i think the icon representing icon-end is bound to change to be a different icon altogether, so i don't know if that's best at the moment

**Pull request merged**